ddl/column.go

```diff
@@ -379,6 +431,20 @@ func doModifyColumn(t *meta.Meta, job *model.Job, newCol *model.ColumnInfo, oldN
 	return ver, nil
 }
 
+// CheckForNullValue ensure there are no null values of the column of this table.
+func CheckForNullValue(ctx sessionctx.Context, schema, table, oldCol, newCol model.CIStr) error {
+	sql := fmt.Sprintf("select count(*) from `%s`.`%s` where `%s` is null;", schema.L, table.L, oldCol.L)
```

Comment:
Do we need to know the number of rows? If not, use `limit 1` is better.

Reply:
limit is enough.